cirq/ops/parity_gates.py

```diff
@@ -159,6 +160,14 @@ class ZZPowGate(eigen_gate.EigenGate,
         where w = e^{i \pi t} and '.' means '0'.
     """
 
+    def _decompose_(self, qubits):
+        # See eigen_gate.EigenGate for more info about the global phase
+        phase = np.exp(1j * np.pi * self.global_shift * self.exponent)
+        yield GlobalPhaseOperation(coefficient=phase)
```

Comment:
This global phase is really multiplied by the exponent?

This could also just be the global phase on the CZPowGate.

Reply:
I am using the information from here:
https://github.com/quantumlib/Cirq/blob/master/cirq/ops/eigen_gate.py#L99

If you don't multiply by the exponent, the test fails for gates that have global_shift.

Reply:
Yes.  My suggestion was to set the global_shift argument to CZPowGate to e.g. `-global_shift/2` instead of using a separate GlobalPhaseOperation.  It's fine the way it is.

Reply:
Oh! Now that I understand what you mean, I think that's a great idea.  I have gone and updated the global phase to be a ghobal_shift on the CZPowGate.